website/docs/zh/config/lib/bundle.mdx

```diff
@@ -1 +1,144 @@
 # lib.bundle
+
+- **类型：** `boolean`
+- **默认值：** `undefined`
+
+指定是否打包库，当 `bundle` 设置为 `true` 时称为打包模式，设置为 `false` 时称为 bundleless 模式。更多详情请参见 [bundle / bundleless](/guide/basic/output-structure#bundle--bundleless)。
+
+::: warning
+无打包模式尚未完全支持，某些功能如 [资源](/guide/advanced/assets) 可能无法正常工作。
+:::
+
+## 设置入口
+
+我们需要为构建指定入口文件。
+
+### bundle: true
+
+当 `bundle` 设置为 `true` 时，入口需要设置为入口文件。默认入口为 `src/index.(ts|js|tsx|jsx|mjs|cjs)`。你需要确保入口文件存在，或通过 [source.entry](https://rsbuild.dev/zh/config/source/entry) 配置自定义入口。
```

Comment:
do we need to update all rsbuild config links from `/config/rsbuild/...`?

Reply:
I have considered doing this, but currently, Rslib's Rsbuild has very little content aside from the redirection. I think we can change the link after we add enough content.